skt/__init__.py

```diff
@@ -531,15 +531,23 @@ def mktgz(self, clean=True, timeout=60 * 60 * 12):
         stdout_list = []
         self.prepare(clean)
 
-        args = self.defmakeargs + ["INSTALL_MOD_STRIP=1",
-                                   "-j%d" % multiprocessing.cpu_count(),
-                                   "targz-pkg"]
+        # Set up the arguments and options for the kernel build
+        targz_pkg_argv = [
+            "INSTALL_MOD_STRIP=1",
+            "-j%d" % multiprocessing.cpu_count(),
+            "targz-pkg"
+        ]
+        kernel_build_argv = (
+            self.make_argv_base +
+            targz_pkg_argv +
```

Comment:
https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Reply:
Will do.